Templates/Inputs/inputs.html

```diff
@@ -5,115 +5,256 @@
+                        </span>
+                        <span class="smol-title">
+                            Images
+                        </span>
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <span class="smol-ico">
+                            <i class="fs-l far fa-keyboard"></i>
+                        </span>
+                        <span class="smol-title">
+                            Input
+                        </span>
+                    </a>
+                </li>
+
+            </ul>
+        </div>
+        <div class="btn-toggle-drawer">
+            <span class="ico closed-ico">
+                <i class="fas fa-chevron-circle-right"></i>
+            </span>
+            <span class="ico open-ico">
+                <i class="fas fa-chevron-circle-left"></i>
+            </span>
+        </div>
+    </div>
+
+    <div class="container">
+        <h1 class="pd-y-md mg-y-sm"> Inputs
+        </h1>
+
+        <div class="contentArticle pd-y-sm">
+            Inputs are means of getting data from user. So, in a sense, it can be said inputs are also a way for user to
+            interact with the interface of the website.
+        </div>
+        <h2 class="contentHeader mg-y-sm">Text Input fields</h2>
+        <div class="contentArticle pd-y-sm">
+            These are the basic input areas, and the basic necessity to get user to input custom data and not select an
+            answer from predefined options
+        </div>
+
+        <div class="contentInput">
+            <div class="boxInput mg-y-sm">
+                <label for="name" class="boxLabel">Name</label>
+                <input type="text" id="name" class="itemInput fs-l">
             </div>
-            <div class="btn-toggle-drawer">
-                <span class="ico closed-ico">
-                    <i class="fas fa-chevron-circle-right"></i>
-                </span>
-                <span class="ico open-ico">
-                    <i class="fas fa-chevron-circle-left"></i>
-                </span>
+            <div class="boxInput">
+                <label for="password" class="boxLabel">Password</label>
+                <input type="password" id="password" class="itemInput fs-l">
             </div>
         </div>
-    
-        <div class="container">
-            
+
+        <pre>
+                <code class="html">
+        &lt;div class="contentInput"&gt;
+            &lt;div class="boxInput mg-y-sm"&gt;
+                &lt;label for="name" class="boxLabel"&gt;Name&lt;/label&gt;
+                &lt;input type="text" id="name" class="itemInput fs-l"&gt;
+            &lt;/div&gt;
+            &lt;div class="boxInput"&gt;
+                &lt;label for="password" class="boxLabel"&gt;Password&lt;/label&gt;
+                &lt;input type="password" id="password" class="itemInput fs-l"&gt;
+            &lt;/div&gt;
+        &lt;/div&gt;
```

Comment:
You can use JS to just replace all the `<` in the pre container with `&LT`, then you can write plain HTML